Dialogs/VX_integration_dialog.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from PyQt5 import uic
+from PyQt5 import QtWidgets
+
+
+class VXDialog(QtWidgets.QDialog):
+    def __init__(self, parent=None):
+        super(VXDialog, self).__init__(parent)
+        uic.loadUi(os.path.join(os.path.dirname(__file__), 'VX_integration_dialog_base.ui'), self)
```

Comment:
Code duplication IMO.
This is the same as previous one. Just initialize from ui and nothing more. Create base class and those two classes should inherit from base. Only difference is name of ui file.

Reply:
Removed dublication